ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -198,8 +198,8 @@ else
 	#
 	# Start system state indicator.
 	#
-	rgbled start -X
-	rgbled_ncp5623c start -X
+	rgbled start -X -q
```

Comment:
We should think about gathering up and centralizing all of these external optional I2C things. Maybe initially we could hide them all behind a single parameter for optionally probing external I2C buses?